lib/aruba.py

```diff
@@ -94,6 +89,13 @@ def _TranslatePolicy(self, pol, exp_info):
       if 'ipv6' in filter_options:
         ip_ver = 6
       new_terms = []
+      for t in terms:
```

Comment:
Yea, some platforms do it out of necessity, and I am sure there are some that do it when it is not needed. This isn't a big deal, more an observation and sparking the idea of a future optimization that we could think about.